src/main/java/org/example/utils/Common.java

```diff
@@ -1,11 +1,24 @@
 package org.example.utils;
 
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
 import java.util.List;
 import java.util.stream.Collectors;
 
 public class Common {
     public static <T> List<T> sort(List<T> list) {
         return list.stream().sorted().collect(Collectors.toList());
     }
+
+    public static boolean isBeforeDate(String date1, String date2) {
```

Comment:
useless method, you can parse two dates directly then use LocalDate API to compare